cli-plugins/plugin/plugin.go

```diff
@@ -14,6 +17,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// CliPluginCloseSocketEnvKey is used to pass the plugin being
+// executed the abstract socket name it should listen on to know
+// when the CLI has exited.
+const CliPluginCloseSocketEnvKey = "CLI_PLUGIN_EXIT_SOCKET"
```

Comment:
```suggestion
const CliPluginCloseSocketEnvKey = "DOCKER_CLI_PLUGIN_EXIT_SOCKET"
```

Reply:
Changed name! Now:

```go
const CLIPluginSocketEnvKey = "DOCKER_CLI_PLUGIN_SOCKET"
```

Reply:
Works for me! Keeping it more generic, so that we could still decide to use the socket for actual communication in future 👍